src/SearchResponse.js

```diff
@@ -31,14 +29,14 @@ class SearchResponse extends React.Component {
   }
 
   responseUpdated = () => {
-    this.setState({ response: pipeline.getResponse() });
+    this.setState({ response: this.props.pipeline.getResponse() });
   };
 
   render() {
-    const { config, tabsFilter } = this.props;
+    const { config, tabsFilter, pipeline, values } = this.props;
     const { response } = this.state;
 
-    if (response.isEmpty() || !response.getQueryValues().q) {
+    if (response.isEmpty() || !response.getQueryValues().q || !values.get().q) {
```

Comment:
What about `response.isError()`?

Why do you need to check both `response.getQueryValues()` and `values`?